### PR TITLE
Create api for getting a user's subscriptions #46

### LINKED ISSUE #46: Create API for getting a user's subscriptions.
GET "/users/{id}/subscriptions?active=<true/false>"

Return subscriptions for the specified user applying the `active` Boolean parameter. ALL if the parameter is null, and so on.

### PR DESCRIPTION
**Pull request by andy-sam**
fixes #46

### CODE REVIEW

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 1]
```diff
@@ -19,3 +19,5 @@ databaseChangeLog:
       file: db/changelog/changelog-2021-07-14-subscription.yaml
   - include:
       file: db/changelog/changelog-2021-07-16-bankServises.yaml
+  - include:
+      file: db/changelog/changelog-2021-07-20-SubscriptionCenceling.yaml
```
Comment (alex-anakin) [inline src/main/resources/db/changelog/db.changelog-master.yaml:23]: empty line

File: src/main/java/com/codesarge/banking/controllers/UserController.java [hunk 2]
```diff
@@ -59,4 +62,14 @@ public void deleteUser(@PathVariable("id") Long id) {
         log.info("Deleting user " + id + " requested");
         userService.deleteUser(id);
     }
+
+    @GetMapping("users/{id}/subscriptions")
+    @ResponseBody
+    public List<SubscriptionDto> findAllSubscriptionsByUserId(
+            @PathVariable("id") Long id,
+            @RequestParam(required = false) Boolean active) {
+        log.info("Subscriptions list for user " + id + " is requested");
```
[the remark this answers is not in the document]
Comment (andy-sam) [reply src/main/java/com/codesarge/banking/controllers/UserController.java:71]: done

File: src/main/resources/db/changelog/db.changelog-master.yaml (hunk 1 above)
Comment (andy-sam) [reply src/main/resources/db/changelog/db.changelog-master.yaml:23]: done

File: src/main/java/com/codesarge/banking/controllers/SubscriptionController.java [hunk 3]
```diff
@@ -35,4 +39,17 @@ public SubscriptionController(
         }
         return ResponseEntity.ok(responseDto);
     }
+
+    @DeleteMapping("bank/services/subscriptions")
+    @ResponseBody
+    public ResponseEntity<Subscription> cencelSubscription(@RequestParam Long accountId,
```
[the remark this answers is not in the document]
Comment (andy-sam) [reply src/main/java/com/codesarge/banking/controllers/SubscriptionController.java:45]: done

File: src/main/java/com/codesarge/banking/controllers/SubscriptionController.java [hunk 4]
```diff
@@ -35,4 +39,17 @@ public SubscriptionController(
         }
         return ResponseEntity.ok(responseDto);
     }
+
+    @DeleteMapping("bank/services/subscriptions")
```
[the remark this answers is not in the document]
Comment (andy-sam) [reply src/main/java/com/codesarge/banking/controllers/SubscriptionController.java:43]: done

**Pull request closed without merging**